libmscore/textline.cpp

```diff
@@ -38,13 +60,51 @@ static const ElementStyle textLineStyle {
       { Sid::textLinePosAbove,                   Pid::OFFSET                  },
       };
 
+//---------------------------------------------------------
+//   systemTextLineStyle
+//---------------------------------------------------------
+
+static const ElementStyle systemTextLineStyle {
+//       { Sid::systemTextLineSystemFlag,           Pid::SYSTEM_FLAG             },
```

Comment:
Does this really need to be commented out? I thought you used it in several places

Reply:
The flag is set at another place since the textline can now be both system and non-system.